src/test/resources/features/mpam/TeamRouting/Campaigns.feature

```diff
@@ -12,21 +12,21 @@ Feature: Campaigns
     Then the case should be moved to the "Campaign" stage
     And the case should be in the correct MPAM "Campaign" team workstack
     Examples:
-      | businessArea | refType       | stage  |
-      | UKVI         | Ministerial   | Triage |
-      | BF           | Ministerial   | Triage |
-      | IE           | Ministerial   | Triage |
-      | EUSS         | Ministerial   | Triage |
-      | HMPO         | Ministerial   | Triage |
-      | Windrush     | Ministerial   | Triage |
-      | Coronavirus  | Ministerial   | Triage |
-      | UKVI         | Official      | Triage |
-      | BF           | Official      | Triage |
-      | IE           | Official      | Triage |
-      | EUSS         | Official      | Triage |
-      | HMPO         | Official      | Triage |
-      | Windrush     | Official      | Triage |
-      | Coronavirus  | Official      | Triage |
+      | businessArea           | refType     | stage  |
+#      | UKVI                   | Ministerial | Triage |
```

Comment:
Do these comments need to be removed?

Reply:
Must have missed them in the purge - will fix now